examples/js/math/Octree.js

```diff
@@ -14,6 +14,8 @@
 
 	const _capsule = new THREE.Capsule();
 
+	const _epsilon = new THREE.Vector3( 0.01, 0.01, 0.01 );
```

Comment:
How have you come up with a value of `0.01`? 

This value is probably way to high for small scaled scenes.

Reply:
You are right, for small scaled scenes the value is way too high. We are using units in millimeters, so our scene is scaled by some factor.
Since a suitable value is dependant on the scene scale, would it be an idea to add `_epsilon` as an argument to the constructor? And what would be a proper default value?

Reply:
I guess I would start with the smallest possible epsilon which is `Number.EPSILON`.

Alternatively, `fromGraphNode()` could be enhanced similar to what `ConvexHull` does. Meaning it uses the input data to compute the extends along all three axes and then uses the max/min values to compute an optimal epsilon.